server/tables/pgcatalog/pg_statistic_ext_data.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgcatalog
+
+import (
+	"io"
+
+	"github.com/dolthub/go-mysql-server/sql"
+
+	"github.com/dolthub/doltgresql/server/tables"
+	pgtypes "github.com/dolthub/doltgresql/server/types"
+)
+
+// PgStatisticExtDataName is a constant to the pg_statistic_ext_data name.
+const PgStatisticExtDataName = "pg_statistic_ext_data"
+
+// InitPgStatisticExtData handles registration of the pg_statistic_ext_data handler.
+func InitPgStatisticExtData() {
+	tables.AddHandler(PgCatalogName, PgStatisticExtDataName, PgStatisticExtDataHandler{})
+}
+
+// PgStatisticExtDataHandler is the handler for the pg_statistic_ext_data table.
+type PgStatisticExtDataHandler struct{}
+
+var _ tables.Handler = PgStatisticExtDataHandler{}
+
+// Name implements the interface tables.Handler.
+func (p PgStatisticExtDataHandler) Name() string {
+	return PgStatisticExtDataName
+}
+
+// RowIter implements the interface tables.Handler.
+func (p PgStatisticExtDataHandler) RowIter(ctx *sql.Context) (sql.RowIter, error) {
+	// TODO: Implement pg_statistic_ext_data row iter
+	return emptyRowIter()
+}
+
+// Schema implements the interface tables.Handler.
+func (p PgStatisticExtDataHandler) Schema() sql.PrimaryKeySchema {
+	return sql.PrimaryKeySchema{
+		Schema:     pgStatisticExtDataSchema,
+		PkOrdinals: nil,
+	}
+}
+
+// pgStatisticExtDataSchema is the schema for pg_statistic_ext_data.
+var pgStatisticExtDataSchema = sql.Schema{
+	{Name: "stxoid", Type: pgtypes.Oid, Default: nil, Nullable: false, Source: PgStatisticExtDataName},
+	{Name: "stxdinherit", Type: pgtypes.Bool, Default: nil, Nullable: false, Source: PgStatisticExtDataName},
+	{Name: "stxdndistinct", Type: pgtypes.Text, Default: nil, Nullable: true, Source: PgStatisticExtDataName},    // TODO: pg_ndistinct type, collation C
+	{Name: "stxddependencies", Type: pgtypes.Text, Default: nil, Nullable: true, Source: PgStatisticExtDataName}, // TODO: pg_dependencies type, collation C
+	{Name: "stxdmcv", Type: pgtypes.Text, Default: nil, Nullable: true, Source: PgStatisticExtDataName},          // TODO: pg_mcv_list type, collation C
+	{Name: "stxdexpr", Type: pgtypes.TextArray, Default: nil, Nullable: true, Source: PgStatisticExtDataName},    // TODO: pg_statistic[] type
```

Comment:
Not sure `Text` is right to use for these in the meantime

Reply:
It's probably fine. I'd be shocked if this table is required for any client to work atm.